src/packages/auth-ui-components/src/components/secret/component.tsx

```diff
@@ -0,0 +1,50 @@
+import { Field, useFormikContext } from 'formik';
+import styles from './styles.module.css';
+import { Button, CustomFieldArgs, PanelMode } from '@exogee/graphweaver-admin-ui-components';
+import { useState } from 'react';
+
+export const SecretFieldComponent = (args: CustomFieldArgs) => {
+	const [hasGeneratedSecret, setHasGeneratedSecret] = useState(false);
+	const { setFieldValue } = useFormikContext();
+
+	const isCreatePanel = args.panelMode === PanelMode.CREATE;
+
+	const handleGenerateSecret = async () => {
+		// Generate secret
+		await (async function () {
```

Comment:
The wrapping here can be removed. So remove this line and line 22 and it all should still work.